packages/ppx/test/native/At_rule_test.re

```diff
@@ -5,87 +5,200 @@ let loc = Location.none;
+    [%expr CssJs.style([|CssJs.media({js|(min-width: 33px) |js}, [||])|])],
   ),
   (
-    "screen and (min-width: 33px) or (max-height: 15rem)",
-    [%expr
-      [%cx "@media screen and (min-width: 33px) or (max-height: 15rem) {}"]
-    ],
+    "(min-width: 33px)",
+    [%expr [%cx "@media (min-width: 33px), (min-width: 13px) {}"]],
     [%expr
       CssJs.style([|
-        CssJs.media(
-          {js|screen and (min-width: 33px) or (max-height: 15rem)|js},
-          [||],
-        ),
+        CssJs.media({js|(min-width: 33px), (min-width: 13px) |js}, [||]),
       |])
     ],
   ),
+  (
+    "(width >= 33px)",
+    [%expr [%cx "@media (width >= 33px) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(width >= 33px) |js}, [||])|])],
+  ),
+  (
+    "(width <= 33px)",
+    [%expr [%cx "@media (width <= 33px) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(width <= 33px) |js}, [||])|])],
+  ),
+  (
+    "(width < 33px)",
+    [%expr [%cx "@media (width < 33px) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(width < 33px) |js}, [||])|])],
+  ),
+  (
+    "(width > 33px)",
+    [%expr [%cx "@media (width > 33px) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(width > 33px) |js}, [||])|])],
+  ),
+  (
+    "(30em <= width <= 50em)",
+    [%expr [%cx "@media (30em <= width <= 50em) {}"]],
+    [%expr
+      CssJs.style([|CssJs.media({js|(30em <= width <= 50em) |js}, [||])|])
+    ],
+  ),
+  (
+    "(30em < width < 50em)",
+    [%expr [%cx "@media (30em < width < 50em) {}"]],
+    [%expr
+      CssJs.style([|CssJs.media({js|(30em < width < 50em) |js}, [||])|])
+    ],
+  ),
   (
     "(hover: hover)",
     [%expr [%cx "@media (hover: hover) {}"]],
-    [%expr CssJs.style([|CssJs.media({js|(hover: hover)|js}, [||])|])],
+    [%expr CssJs.style([|CssJs.media({js|(hover: hover) |js}, [||])|])],
+  ),
+  (
+    "print",
+    [%expr [%cx "@media print {}"]],
+    [%expr CssJs.style([|CssJs.media({js|print |js}, [||])|])],
+  ),
+  (
+    "not print",
+    [%expr [%cx "@media not print {}"]],
+    [%expr CssJs.style([|CssJs.media({js|not print |js}, [||])|])],
+  ),
+  (
+    "screen and (color))",
+    [%expr [%cx "@media screen and (color) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|screen and (color) |js}, [||])|])],
+  ),
+  (
+    "(screen)",
+    [%expr [%cx "@media (screen) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(screen) |js}, [||])|])],
+  ),
+  (
+    "screen, print",
+    [%expr [%cx "@media screen, print {}"]],
+    [%expr CssJs.style([|CssJs.media({js|screen, print |js}, [||])|])],
+  ),
+  (
+    "(screen, print))",
+    [%expr [%cx "@media (screen, print) {}"]],
+    [%expr CssJs.style([|CssJs.media({js|(screen, print) |js}, [||])|])],
+  ),
+  (
+    "(screen and (color))",
+    [%expr [%cx "@media (screen and (color)) {}"]],
+    [%expr
+      CssJs.style([|CssJs.media({js|(screen and (color)) |js}, [||])|])
+    ],
+  ),
+  (
+    "(not (screen and (color))))",
```

Comment:
That's a weird example:

Usually this is used: `not screen and (color)`

Reply:
Thanks, I will add a better example, but also keep the other